copilot/fastapi/tests/test_main.py

```
@@ -0,0 +1,44 @@
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)